src/ostree/ot-builtin-commit.c

```diff
@@ -147,13 +149,68 @@ parse_statoverride_file (GHashTable   **out_mode_add,
   return ret;
 }
 
+/* XXX: share more code with parse_statoverride_file() */
+static gboolean
+parse_skiplist_file (GHashTable   **out_skiplist,
+                     GCancellable  *cancellable,
+                     GError        **error)
+{
+  gboolean ret = FALSE;
+  gsize len;
+  char **iter = NULL; /* nofree */
+  g_autoptr(GHashTable) ret_hash = NULL;
+  g_autoptr(GFile) path = NULL;
+  g_autofree char *contents = NULL;
+  char **lines = NULL;
+
+  path = g_file_new_for_path (opt_skiplist_file);
+
+  if (!g_file_load_contents (path, cancellable, &contents, &len, NULL, error))
+    goto out;
+
+  ret_hash = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, NULL);
+  lines = g_strsplit (contents, "\n", -1);
+
+  for (iter = lines; iter && *iter; iter++)
+    {
+      const char *line = *iter;
+
+      if (*line == '-')
```

Comment:
My memory is a bit dim on why the statoverride files start with `+`...maybe so they could have comments with `#`?  Dunno.  If so I doubt it was used.

It seems like it'd be simpler to just have a newline-separated list.  (Anyone who complains about this should really be using the API IMO)